revisions-extended/src/utils.js

```diff
@@ -22,11 +22,11 @@ export const getEditUrl = ( postId ) => {
 };
 
 export const getAllRevisionUrl = ( type ) => {
-	if ( type.toLowerCase() === 'page' ) {
-		return '/wp-admin/edit.php?page=page-updates';
+	if ( type.toLowerCase() === 'post' ) {
+		return '/wp-admin/edit.php?page=post-updates';
 	}
 
-	return '/wp-admin/edit.php?page=post-updates';
+	return `/wp-admin/edit.php?post_type=${ type }&page=page-updates`;
```

Comment:
This should actually be `&page=${ type }-updates`. Though, that's kind of redundant since there's also a `post_type` query parameter. I guess I could shorten the page slug to just `updates`... 🤔

Reply:
Ah okay, I just navigated there using the admin link and copy & pasted it.

Reply:
Er, I just discovered that he page slug has to be unique across all menus, even though that’s not what it says in the function docs. If it’s just `updates`, and you’re looking at the updates for pages, the menu still shows that you’re looking at updates for posts. So I guess we need to leave it as `&page=${ type }-updates`